.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Stuff like this should go in your personal .gitignore rather than the one in the repo:

```bash
$ git config --global core.excludesFile ~/.gitignore
$ echo .idea >> ~/.gitignore
```